corehq/form_processor/tests/test_forms.py

```diff
@@ -428,6 +441,30 @@ def test_nothing_is_deleted_if_dry_run_is_true(self):
         # still returns accurate count
         self.assertEqual(counts, {'form_processor.XFormInstance': 1})
 
+    def test_tombstone_is_created_on_deletion(self):
+        form = create_form_for_test(self.domain, deleted_on=datetime(2020, 1, 1, 12, 29))
+
+        XFormInstance.objects.hard_delete_forms_before_cutoff(self.cutoff, dry_run=False)
+        delete_doc = DeletedSQLDoc.objects.filter(doc_id=form.form_id)
+
+        self.assertIsNotNone(delete_doc)
+
+    def test_tombstone_count_matches_deleted_form_count(self):
+        expected_count = 5
+        for _ in range(expected_count):
+            create_form_for_test(self.domain, deleted_on=datetime(2020, 1, 1, 12, 29))
+
+        counts = XFormInstance.objects.hard_delete_forms_before_cutoff(self.cutoff, dry_run=False)
+
+        self.assertEqual(DeletedSQLDoc.objects.all().count(), counts['form_processor.XFormInstance'])
+
+    def test_tombstone_is_not_create_if_deleted_on_is_null(self):
+        create_form_for_test(self.domain, deleted_on=None)
+
+        XFormInstance.objects.hard_delete_forms_before_cutoff(self.cutoff, dry_run=False)
+
+        self.assertFalse(list(DeletedSQLDoc.objects.all()))
```

Comment:
super nit (I just think it is more readable)
```suggestion
        self.assertEqual(DeletedSQLDoc.objects.count(), 0)
```